strings/junkmail.txt

```diff
@@ -13,3 +13,17 @@ MONTHPEOPLE ARE REAL, AND NO I DON'T MEAN "MOTH" PEOPLE! CALENDAR CRYPTIDS! THE
+I NEED HELP RIGHT NOW, AS WE SPEAK THE COOK IS DEEPFRYING EVERY SINGLE ONE OF MY POSESSIONS. IT STARTED WHEN AN ASSISTANT SHOVED ME OVER AND HANDCUFFED ME, THEY JOKINGLY STARTED WRAPPING EVERY SINGLE ONE OF MY STUFF IN THIS WRAPPING PAPER. I BEGGED FOR HELP AND A COOK CAME TO HELP, BUT THEY SAW THE PACKED BOXES AND STARTED FRYING THEM. I'M CRYING AND YELLING AND I DON'T KNOW HOW TO GET THEM TO STOP, PLEASE HELP ME!
+"Law 2" this, "Law 2" that, how about you "Law 2" get yourself an actual job, then maybe you'll actually have access to all those doors you ask to be opened. Or better yet, maybe you'll actually be a respectable employee of the station and be let in. Greytider.
+C*ders are out to destroy everything. I can't send you this letter without censoring the full word because otherwise they'll catch on and find us. I need you to send me 125 credits back to fund the resistance, we need to take back what they have taken from us. If you never hear from me again, assume I am dead and my body will never be found. Please, help, and get the credits sent as soon as possible. Signed, V. Iva
+Hello this is the moderator from Cargo Chatroom 'Bloodsucker Roleplay Group'. We have noticed you haven't logged in for 4 weeks, we are just checking to see that everything is okay with our biggest fan. Since you visited us last time we have updated the rules to allow for better and smoother roleplay, hope to see you back soon!
+I don't really get the over-reliance on security. Basically sec claims day 1 and confirms a lot of the crew which already tilts the station in favour of Nanotrasen. Even after that if two people fight for the same job and say each other is fake then the station can afford to actually lynch a crewmember because security can execute the other the same night. Also literally every security member claims day 1 and uses the same strategy making it extremely boring. I recently started working on another station and I can already notice a much improved quality in stations without the security department. People are actually creative and use new strategies instead of just copying existing ones. But even there if there's security then the station is just as boring again. I think the station would be much better off without this department.
```

Comment:
```suggestion
I don't really get the over-reliance on security. Basically sec claims day 1 and confirms a lot of the crew which already tilts the station in favour of Nanotrasen. Even after that if two people fight for the same job and say each other is fake then the station can afford to actually lynch a crewmember because security can execute the other the same night. Also literally every security member claims day 1 and uses the same strategy making it extremely boring. I recently started working on another station and I can already notice a much improved quality in stations without the security department. People are actually creative and use new strategies instead of just copying existing ones. But even there if there's security then the station is just as boring again. I think the station would be much better off without this department.
Sending mail is my life. I die for my Mail Sendoomfies. It's some real shit. I meet the love of my life here, I mature, it's part of the Sending Mail grindset. 🗣️💯
```

Reply:
rename goth to something SS13 related and I'll do it

Reply:
JANIBORG GF

Reply:
done